controllers/operandbindinfo/operandbindinfo_controller.go

```diff
@@ -400,6 +427,173 @@ func (r *Reconciler) copyConfigmap(ctx context.Context, sourceName, targetName,
 	return false, nil
 }
 
+func (r *Reconciler) copyRoute(ctx context.Context, route operatorv1alpha1.Route, targetName, sourceNs, targetNs, key string,
+	bindInfoInstance *operatorv1alpha1.OperandBindInfo, requestInstance *operatorv1alpha1.OperandRequest) (requeue bool, err error) {
+	if route.Name == "" || sourceNs == "" || targetNs == "" {
+		return false, nil
+	}
+
+	if route.Name == targetName && sourceNs == targetNs {
+		return false, nil
+	}
+
+	if targetName == "" {
+		if publicPrefix.MatchString(key) {
+			targetName = bindInfoInstance.Name + "-" + route.Name
```

Comment:
Please add the resource type `route` into the targetName, to differentiate between ConfigMap for Route vs Service in case the underlaying resources have the same name